azext_edge/edge/providers/rpsaas/adr/asset_endpoint_profiles.py

```diff
@@ -442,3 +418,18 @@ def _update_properties(
             username_reference=username_reference,
             password_reference=password_reference
         )
+
+
+def _validate_additional_configuration(schema: dict, config: dict):
+    from jsonschema import Draft7Validator, ValidationError
+    validator = Draft7Validator(schema=schema)
+
+    errors = []
+    error: ValidationError
+    for error in validator.iter_errors(config):
+        errors.append(f"- argument {error.schema['description']}: {error.message}")
```

Comment:
nit, "- argument _for_ ..." to delineate the argument is for the parameter not the param itself.